test/user.test.js

```diff
@@ -16,14 +18,21 @@ describe('User', function() {
   var invalidCredentials = {email: 'foo1@bar.com', password: 'invalid'};
   var incompleteCredentials = {password: 'bar1'};
 
+  var defaultApp;
+
   beforeEach(function() {
-    User = loopback.User.extend('user');
+    // FIXME: [rfeng] Remove loopback.User.app so that remote hooks don't go
+    // to the wrong app instance
+    defaultApp = loopback.User.app;
+    loopback.User.app = null;
+    User = loopback.User.extend('TestUser', {}, {http: {path: 'test-users'}});
```

Comment:
Anyways. If you believe that it's better to have `/test-users` URL even at the cost of making future code archaeology more difficult, then I can live with that.